### ISSUE TITLE
Refactor code

### ISSUE THREAD

**Issue by H-Anna** [H-Anna/Clover#12]
The code needs to be refactored according to C++ style guide tips as described here.
First it needs to be decided which style points should be enforced. Then they should be implemented, at a time when no new features are being worked on.
This is low priority. #11 and #10 should be implemented first, however refactoring should happen before certain goals in #4 such as cross-ghost talk.
